src/intervals/rounding.jl

```diff
@@ -9,6 +9,7 @@ interval rounding types, e.g.
 +(IntervalRounding{:none}, a, b, RoundDown)
 
 The current allowed rounding types are
+- :errorfree # use errorfree arithmetic via FastRounding.jl
```

Comment:
The problem is that this implies that `:fast` is not `:tight`, which is incorrect. (Errorfree arithmetic does give tight bounds.)